user/cgiwifi.c

```diff
@@ -485,6 +487,11 @@ int ICACHE_FLASH_ATTR printWifiInfo(char *buff) {
 
 	uint8_t op = wifi_get_opmode() & 0x3;
 	char *mode = wifiMode[op];
+#ifdef CHANGE_TO_STA
+	char *modechange = "yes";
```

Comment:
This will produce a variable in RAM with the text. How about just having those chars in flash? Maybe make this `static const char *`?